src/App.tsx

```diff
@@ -1,27 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
+  const handleLoadAllGoodsClick = async () => setGoods(await getAll());
+  const handleLoad5FirstGoodsClick = async () => setGoods(await get5First());
+  const handleLoadRedGoodsClick = async () => setGoods(await getRedGoods());
```

Comment:
you can also create one general function that will accept needed handler and will call it